src/components/Birthday.tsx

```diff
@@ -0,0 +1,29 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import {useState} from "react";
+import "./birthday.scss";
+
+const Birthday = () => {
```

Comment:
The name of the component is not clear by its meaning. 

I would recommend to name it "BirthdayPicker", "BirthdayInput", "DayPicker", "DaySelector", etc.